packages/ag-charts-community/src/chart/series/cartesian/areaSeriesModule.ts

```diff
@@ -0,0 +1,59 @@
+import type { SeriesModule } from '../../../util/module';
```

Comment:
Just an FIY: this type is moving into _coreModules.ts_ in PR #222; so the build errors / merge-conflicts will need to be resolved by someone.